_posts/2021-05-21-pytesting-in-openfaas.md

```diff
@@ -0,0 +1,331 @@
+---
+title: "Testing Python Functions with Pytest"
+description: "Just because you can test in Prod doesn't mean you have to"
```

Comment:
FYI "Just because you can test in Prod doesn't mean you have to" is what will be used for SEO, so you may want to move this comment into the body somewhere in context and write more of a short summary here instead.

Reply:
what do you think about 

> In this post we show you how to integrate Pytest into your OpenFaaS workflow. When you use the python-flask-template these tests are automatically integrated into the build process and your CI/CD workflow."